public_html/profiles/os2web/modules/os2web_accessibility/os2web_accessibility.module

```diff
@@ -265,7 +267,7 @@ function os2web_accessibility_site_map_menus($true = FALSE) {
 function os2web_accessibility_menu_link_add_handicap($tree = array()){
   foreach ($tree as $i =>$link) {
     foreach ($link as $key => $value) {
-      foreach ($value as $name => $v) {
+      foreach (@$value as $name => $v) {
```

Comment:
it confusing why we need a suppress sign here.
Maybe add a comment or make the code more readable?

Reply:
@stanbellcom if menu punkt have no child punkts when "below" item is FALSE, and warning appeared. I changed code a little https://github.com/syddjurs/syddjurs.dk/pull/169/commits/2670201384e129a508e11056f9de471e934b6e1d#diff-451abd137472af0eebe44aa8a5c4075cR271